server/Controllers/OrderController.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.IdentityModel.Tokens;
+using server.DTO;
+using server.Models.DB;
+using server.Services;
+using System;
+
+namespace server.Controllers
+{
+    [Route("[controller]")]
+    [ApiController]
+    public class OrderController : ControllerBase
+    {
+        private readonly IOrderService _orderService;
+
+        public OrderController(IOrderService orderService)
+        {
+            _orderService = orderService;
+        }
+
+        [HttpGet]
+        public async Task<IActionResult> Get()
```

Comment:
Method names can be more verbose , eg : GetAllOrders() in this case. 
Please update the necessary method names so that code readability increases.

Reply:
Made more verbose as suggested